astroplan/plots/time_dependent.py

```diff
@@ -145,40 +148,47 @@ def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
                       'to use a scalar. (Or maybe a list with length > 1?).',
                       PlotWarning)
 
-    # Calculate airmass
-    airmass = observer.altaz(time, target).secz
-    # Mask out nonsense airmasses
-    masked_airmass = np.ma.array(airmass, mask=airmass < 1)
+    if not isinstance(targets, Sequence):
+        targets = [targets]
 
-    # Some checks & info for labels.
-    if not hasattr(target, 'name'):
-        target_name = ''
-    else:
-        target_name = target.name
-    style_kwargs.setdefault('label', target_name)
+    for target in targets:
+        # Calculate airmass
+        airmass = observer.altaz(time, target).secz
+        # Mask out nonsense airmasses
+        masked_airmass = np.ma.array(airmass, mask=airmass < 1)
 
-    # Plot data.
-    ax.plot_date(time.plot_date, masked_airmass, **style_kwargs)
+        # Some checks & info for labels.
+        if not hasattr(target, 'name'):
+            target_name = ''
+        else:
+            target_name = target.name
 
-    # Format the time axis
-    date_formatter = dates.DateFormatter('%H:%M')
-    ax.xaxis.set_major_formatter(date_formatter)
-    plt.setp(ax.get_xticklabels(), rotation=30, ha='right')
+        # Plot data.
+        ax.plot_date(time.plot_date, masked_airmass, label=target_name, **style_kwargs)
+
+        # Format the time axis
+        date_formatter = dates.DateFormatter('%H:%M')
```

Comment:
Why is this line and the following 2 inside the `for target in targets` loop? Wouldn't it suffice to do them once after the looping?